romanisim/bandpass.py

```diff
@@ -129,3 +133,26 @@ def get_abflux(bandpass):
         abflux = compute_abflux()
         get_abflux.abflux = abflux
     return abflux[bandpass]
+
+def compute_count_rate(thru=None, bandpass=None, filename=None, effarea=None):
+    # ab_zero = get_abflux(bandpass)
+
+    if not effarea:
+        effarea = read_gsfc_effarea(filename)
+
+    integrand = thru * constants.c / (
```

Comment:
Note that here 'thru' can only be a constant or something that is compatible with effarea; the old version of this was not a very generic count rate calculator.